mlrun/api/utils/projects/followers/base.py

```diff
@@ -0,0 +1,51 @@
+import abc
```

Comment:
`mlrun/api/utils/projects/followers/base.py` -> `mlrun/api/utils/projects/members/follower.py`

I suggest to put all members in the same dir, there's no need to separate, same mechanism, and it's weird that followers have a dir, but the leader is under member so, would be more tidy that way